Chrome Pickup Sort Fix

searchingfor opened the pull request:
Couple things: 
1. I don't know if doing the sort within the component is bad practice/not what you want.
2. For the line  'return this.pickups.slice(0).sort(compare)', I added the slice(0) because the linter would not accept it without me doing that. I don't know a better solution or if if this is even doing anything.

OK, Cheers 👍 

closes #873

tiltec commented on src/store/modules/pickups.js:
```diff
@@ -239,6 +239,8 @@ export function isWithinOneWeek (pickup) {
 }
 
-export function sortByDate (a, b) {
-  return a.date > b.date
```
Would it be sufficient to replace the line with this:
```
return a.date - b.date;
```

It seems that `a.date > b.date` behaved wrong because it returned true or false instead of integers.

searchingfor commented:
Interestingly, if I don't use a sort function at all in modules/pickup.js (after I have removed the one I added), it is now magically sorted in chrome and firefox? Any idea why this is? @tiltec

tiltec commented:
@searchingfor The backend return the pickups as sorted list, so that should work out of the box. The additional sorting becomes important when handling with data updates while the page is open, for example a change in pickup time or the creation of a new pickup.

tiltec commented:
Great, that looks pleasing to my eyes and seems to work fine :) Thanks @searchingfor!
